sbi/inference/snpe/snpe_c.py

```diff
@@ -508,9 +487,7 @@ def _automatic_posterior_transformation(
         return logits_pp, means_pp, precisions_pp, covariances_pp
 
     def _precisions_proposal_posterior(
-        self,
-        precisions_p: Tensor,
-        precisions_d: Tensor,
+        self, precisions_p: Tensor, precisions_d: Tensor,
```

Comment:
Didn't we recently run `black` over the whole code base? Do you know why these diffs are showing up again?

Reply:
yes, we did. and no, I don't know. I will check my black settings and check again.

Reply:
back then I ran `black` and `isort` from the terminal. It seems the settings are slightly different when using the `vscode` settings, e.g., when it comes to line breaks in function signatures. This is why we get these small changes on some files now.